### ISSUE TITLE
Only getting a few tables in 3.1.0

### ISSUE THREAD

**Issue by agentKnipe** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#557]
Visual Studio updated me to 3.1.0 and I am not only generating 5 of my tables and 3 of my stored procedures? Could this be an issue with the license? I believe I am still using the beta license and its was good till the end of this year.  Figured this would be fine as long it nothing has gone into production.  

thanks

**Comment by sjh37** [comment]
Ah yes, for the trial license, I've set a 5 table and 1 stored proc max limit on it. I figured that was plenty for a trial to prove it works for you, hence why the beta (which is a trial license type) is only generating a few tables. As a thank you for your help, if you send me an email simon@reversepoco.co.uk and I'll send you a commercial license till the end of Dec.

**Comment by LarryWMSN** [comment]
I recommended this to a customer who is trialing the product and really likes it, but we wasted 4 hours before looking here to figure out what happened. It may be a good idea to list the table/sp names and a comment indicating why they weren't generated.

**Comment by sjh37** [comment]
Hi @LarryWMSN Thanks for your recommendation.

The number of tables are listed on the pricing page the user clicked on to add it to their cart: https://www.reversepoco.co.uk/Pricing

Yes, it would be a good idea to include a comment in the code :-)
I'll open this case so make sure I add a comment..

**Comment by LarryWMSN** [comment]
Yeah, we noticed that on the pricing page that after we found this thread. :)

**Comment by sjh37** [comment]
Thanks @LarryWMSN All done. This will be in the next release.
It adds a comment at the top of each `#region`

```c#
// ****************************************************************************************************
// This is not a commercial licence, therefore only a few tables/views/stored procedures are generated.
// ****************************************************************************************************
```

**Comment by sjh37** [comment]
Now released as part of v3.2.0
